docs/docs-beta/docs/guides/quality-testing/unit-tests-assets-and-ops.md

```diff
@@ -4,4 +4,115 @@ sidebar_position: 30
 sidebar_label: "Unit testing"
 ---
 
-# Unit tests for assets and ops
+Unit testing is an important tool for verifying that a computation works as intended.
+
+While this is traditionally difficult in the context of data pipelines, Dagster helps simplify this process by allowing you to directly invoke your computations with specific input values and mocked resources to ensure that they transform data as expected.
+
+Unit tests can't fully replace acceptance tests, but *are* capable of catching a large variety of errors with a significantly faster feedback loop.
```

Comment:
Should we add a short description of acceptance tests, or more explicitly state "human review"?

Reply:
done!